prometheus/src/components/TreeNode.tsx

```diff
@@ -383,7 +383,7 @@ function QueryStatus({
       <Typography variant="body2" component="span" sx={{ color: (theme) => theme.palette.grey[500] }}>
         {resultStats.numSeries} result{resultStats.numSeries !== 1 && 's'}
         &nbsp;&nbsp;–&nbsp;&nbsp;
-        {responseTime}ms
+        {Math.round(responseTime as number)}ms
```

Comment:
If responseTime can be undefined, let handle this case too
```suggestion
        {responseTime ? `${Math.round(responseTime)}ms` : "Unknown response time"}
```

Reply:
I'd say "unknown" or "N/A" to keep it a short string on screen

Reply:
> N/A or empty string? Because N/A or unknown without context can be strange, no?

valid point, what about `? ms` or `— ms` then?

Reply:
I prefer ? ms, else we will have: `@ requests - -ms - ...`

Reply:
> I prefer ? ms, else we will have: `34 requests - - ms - ...`

Yes makes sense